technical-reference/reference-technical-articles/technical-doc-xvs-bridge.md

```diff
@@ -0,0 +1,221 @@
+
+<figure><img src="../../.gitbook/assets/dest-bnb.jpg" alt=""><figcaption></figcaption></figure>
+
+To transfer XVS tokens back to the BNB, follow a similar process as mentioned in the earlier send section. You don't need to approve the Bridge contract on the Destination Chain to spend XVS tokens on your behalf. The tokens will be burned on the Destination Chain on your behalf and unlocked and transferred to the receiver's address on the BNB.
+
+## 5. Monitoring Transaction Status <a name="monitoring-transaction-status"></a>
+
+After initiating a token transfer, you should wait for the transaction to confirm. This process may take a few minutes. Once the transaction confirms, you will receive the bridged XVS tokens on the Destination Chain.
+
+## 6. Security and Risks <a name="security-and-risks"></a>
+
+<figure><img src="../../.gitbook/assets/risk.jpg" alt=""><figcaption></figcaption></figure>
+
+### 6.1. Ownership Transfer <a name="ownership-transfer"></a>
+
+- Use the `transferOwnership` method in the XVSBridgeAdmin contract to transfer ownership of the admin contract.
+- Use the `transferBridgeOwnership` method to transfer ownership of the Bridge contract from one contract to another.
+- Ownership control is crucial in case of emergencies or security issues.
+
+### 6.2. Pause and Resume <a name="pause-and-resume"></a>
+
+- The Bridge includes a pause and unpause mechanism. Use the `pause` method to halt the contract's functionality and `unpause` to resume.
+- Pausing is a security measure to prevent further transactions during emergencies or potential attacks.
+- Cross-chain messages that attempt to mint or release tokens to the receiver can be received by the destination bridge. These messages will fail, but they can be retried once the destination bridge has been unpaused.
+
+### 6.3. Limit the Amount of XVS Transfers <a name="limit-the-amount-of-XVS-transfers"></a>
+
+- Example: Limit the maximum XVS transfer to USD 1,000 in one transaction and USD 100,000 in one day. These limits can be adjusted using VIPs.
+
+### 6.4. Transfer Delays <a name="transfer-Delays"></a>
+
+- Configurable delay after XVS transfers to the target network by specifying a minimum number of blocks in the layer zero endpoint configuration.
+
+### 6.5. Token Controller Contract <a name="token-controller-contract"></a>
+
+- Token Controller contract within the XVS token deployed on the target network to blacklist addresses, preventing them from transferring or receiving XVS. Integrated with the ACM.
+
+### 6.6. Cap on Token Minting <a name="cap-on-token-minting"></a>
+
+- Cap on the amount of tokens that can be minted in the destination target network. This feature can be integrated in Token Controller.
+
+### Bridge Model
+
+The XVSProxyOFTDest contract serves as the bridge model. It will be authorized to mint and burn tokens in the destination chain. Limits on these actions will be set by Governance or Guardian.
```

Comment:
```suggestion
The XVSProxyOFTDest contract serves as the bridge model. It will be authorized to mint and burn tokens in the Destination Chain. Limits on these actions will be set by Governance or Guardian.
```